docs/resources/vpc_peering.md

```diff
@@ -228,3 +249,114 @@ This resource depends on CloudAMQP managed VPC identifier, `cloudamqp_vpc.vpc.id
 ## Import
 
 Not possible to import this resource.
+
+## Create VPC Peering with additional firewall rules
+
+To create a VPC peering configuration with additional firewall rules, it's required to chain the [cloudamqp_security_firewall](https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/security_firewall)
+resource to avoid parallel conflicting resource calls. This is done by adding dependency in the firewall resource to the VPC peering resource (`cloudamqp_vpc_peering.vpc_accept_peering`).
```

Comment:
```suggestion
resource to avoid parallel conflicting resource calls. You can do this by making the firewall resource depend on the VPC peering resource (`cloudamqp_vpc_peering.vpc_accept_peering`).
```